recipe/meta.yaml

```diff
@@ -7,20 +7,29 @@ package:
 source:
   git_url: https://github.com/geopandas/geopandas.git
   git_branch: geopandas-cython
+  patches:
+    - geos_header.patch
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt --with-cython
+  script:
+    - python setup.py install --single-version-externally-managed --record record.txt --with-cython  # [not win]
```

Comment:
What is the `[not win]` on this line for?

Reply:
B/c I added a `bld.bat` with all the extra stuff that only Windows needs.